openadapt/app/tray.py

```diff
@@ -168,6 +183,67 @@ def _quit() -> None:
         # for storing toasts that should be manually removed
         self.sticky_toasts = {}
 
+    def stop_download(self) -> None:
```

Comment:
According to me these functions are related to the system tray?
And wont be related to any other functionality in the system. They make more sense to be the property of system tray rather than other file.